docs/user_guide/workloads/run_nvidia.md

```diff
@@ -0,0 +1,263 @@
+---
+id: run_nvidia
+title: Run NVIDIA GPU Scheduling Jobs
+description: How to run generic example of GPU scheduling with Yunikorn.
+keywords:
+ - NVIDIA GPU
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+To know how the GPU scheduling works, please refer to [**Time-Slicing GPUs in Kubernetes | Introduction**](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/gpu-sharing.html#introduction). This page covers ways to enable GPU scheduling in Yunikorn using [**NVIDIA GPU Operator**](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/gpu-operator).
```

Comment:
Thanks @wilfred-s for the clarification. In that case, I think this part needs to be rephrased. The way it's written seems to suggest that enabling time slicing is a requirement for using GPUs, but in reality, it's an optional feature. We should make that clear. I've heard some production ML training use cases that in fact explicitly turned off time slicing since each work unit is heavy enough to fully use a single GPU. @KatLantyss

Reply:
@KatLantyss please check this comment

Reply:
@wilfred-s Sorry for the delay, I will change it recently, and thanks @yuchaoran2011 for the advice.